blobfuse/utilities.cpp

```diff
@@ -427,15 +393,19 @@ int azs_getattr(const char *path, struct stat *stbuf)
         }
         else
         {
-            return -(ENOENT); // -2 = Entity does not exist.
+            syslog(LOG_DEBUG, "Entity %s does not exist.  Returning ENOENT (%d) from get_attr.\n", path, ENOENT);
```

Comment:
For instance; rewriting all of these as:

"getattr call - Entity %s does not exist. Returning ENOENT (%d)"

Reply:
I tried to make the info / warning / error logs readable on their own, but for debug, my thinking was that someone reading the debug log will have the other debug log entries right there, and every method logs when it's first called into.  I tried to add the input path argument (or something equivalent, like mntPath or blob name) to every log statement, to help detangle them if stuff is happening in parallel.

Reply:
I don't think this justifies it :) I can't understand looking at the following log and determine which call it is coming from... Think about the support team who may end up looking at these logs one day

Reply:
Ok, I'm not super convinced, but no harm in adding it :)  I'll get to this tomorrow.